app/models/equipment_model.rb

```diff
@@ -197,6 +197,15 @@ def num_available(start_date, due_date)
     num_available_from_source(start_date, due_date, relevant_reservations)
   end
 
+  def num_available_excluding(start_date, due_date, exclude_reservation)
```

Comment:
For semantics let's use either `excluded_reservation` or `excluded_reservations` (see my [other comment](https://github.com/YaleSTC/reservations/pull/1256/files#r31100383)).